base/precompilation.jl

```diff
@@ -677,7 +705,7 @@ function precompilepkgs(pkgs::Vector{String}=String[];
             n_print_rows = 0
             while !printloop_should_exit
                 lock(print_lock) do
-                    term_size = Base.displaysize_(io)
```

Comment:
This is not needed because `io` is now inferred?

Reply:
correct, `io` is now concrete